src/OssiaTypes.h

```diff
@@ -85,6 +86,51 @@ template<> struct MatchingType<double> {
   }
 };
 
+template<> struct MatchingType<ossia::impulse> {
+    using ofx_type = ossia::impulse;
+    static constexpr const auto val = ossia::val_type::IMPULSE;
+    using ossia_type = ossia::impulse;
+    using ossia_unit = ossia::unit_t;
+
+    static ofx_type convertFromOssia(const ossia::value& v)
+    {
+      return v.get<ossia_type>();
+    }
+
+    static ossia_type convert(ofx_type f)
+    {
+      return ossia::impulse();
+    }
+};
+
+template<> struct MatchingType<std::vector<string> > {
+  using ofx_type = std::vector<string>;
+  static constexpr const auto val = ossia::val_type::TUPLE;
+  using ossia_type = std::vector<ossia::value>;
+  using ossia_unit = ossia::unit_t;
+
+  static ofx_type convertFromOssia(const ossia::value& v)
+  {
+    const ossia_type& t = v.get<ossia_type>();
+    ofx_type _return{};
+    _return.reserve(t.size());
+    for_each(t.begin(), t.end(), [&_return](const ossia::value& value){
```

Comment:
wouldn't  `for(auto& value : t) { _return.push_back(value.get<std::string>()); }` be simpler ?